src/syrupy/__init__.py

```diff
@@ -42,6 +42,14 @@ def pytest_sessionstart(session: Any) -> None:
     session._syrupy.start()
 
 
+def pytest_collection_modifyitems(session: Any, config: Any, items: List[Any]) -> None:
+    session._syrupy._all_items.update(items)
+
+
+def pytest_collection_finish(session: Any) -> None:
```

Comment:
https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_collection_finish

Reply:
If we use https://github.com/Instagram/MonkeyType, can we avoid the "Any" types?